patches/minecraft/net/minecraft/entity/item/EntityItem.java.patch

```diff
@@ -1,6 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/entity/item/EntityItem.java
 +++ ../src-work/minecraft/net/minecraft/entity/item/EntityItem.java
-@@ -38,6 +38,11 @@
+@@ -22,6 +22,7 @@
+ import net.minecraft.util.math.MathHelper;
+ import net.minecraft.util.text.translation.I18n;
+ import net.minecraft.world.World;
++import net.minecraftforge.event.ForgeEventFactory;
```

Comment:
No imports in patches - use fully-qualified names where needed.

Reply:
Cleaned, cheers